test/CadlRanchProjects/azure/core/lro/standard/src/Generated/StandardClient.cs

```diff
@@ -284,7 +288,11 @@ public virtual Operation<ExportedUser> Export(WaitUntil waitUntil, string name,
 
             RequestContext context = FromCancellationToken(cancellationToken);
             Operation<BinaryData> response = Export(waitUntil, name, format, context);
-            return ProtocolOperationHelpers.Convert(response, ExportedUser.FromResponse, ClientDiagnostics, "StandardClient.Export");
+            return ProtocolOperationHelpers.Convert(response, r =>
+            {
+                var resultJsonElement = System.Text.Json.JsonDocument.Parse(r.Content).RootElement.GetProperty("result");
+                return ExportedUser.DeserializeExportedUser(resultJsonElement);
```

Comment:
This two-line delegate looks really ugly. Maybe generate it as a local function?

Reply:
Somethind like:
```cs
ExportedUser DeserializeResult(Response response)
{
    using var document = JsonDocument.Parse(r.Content);
    return ExportedUser.DeserializeExportedUser(document.RootElement.GetProperty("result"));
}
return ProtocolOperationHelpers.Convert(response, DeserializeResult, ClientDiagnostics, "StandardClient.Export");
```